views/partials/docs/moons/getByName.pug

```diff
@@ -0,0 +1,21 @@
+div(class="section")
+  div(class="lable") Method
+  div(class="value delete") get
+div(class="section")
+  div(class="lable") Route
+  div(class="value") http:localhost:2319/moons/{name}
```

Comment:
This route doesn't work

![image](https://user-images.githubusercontent.com/5775716/84069624-d1e83280-a998-11ea-9bf2-84e0087736fc.png)